src/lib/swapper/swappers/CowSwapper/cowExecuteTrade/cowExecuteTrade.test.ts

```diff
@@ -57,7 +42,34 @@ jest.mock('../utils/helpers/helpers', () => {
   }
 })
 
-const ethereumMock = jest.mocked(ethereum)
+const actualChainAdapters = jest.requireActual('@shapeshiftoss/chain-adapters')
+
+const mockEthereum = {
```

Comment:
Naming nitpick, won't block the PR for that but might be worth appending the `chainAdapter` vernacular here, since `mockEthereum` and `mockGnosis` make it sound like these are mocking assets (which really would be `mockETH` and `mockXDAI` if that was the case, but while scrolling through, that might be the first thing that comes to mind, always better to ease the cognitive load by being descriptive and avoid confusion)

Reply:
Isn't the opposite true here? We're mocking a chain adapter, not a currency

Reply:
Precisely why we need the `chainAdapter` vernacular to avoid confusion.

If you grep for `mockEthereum` around the codebase, you'll see what I mean, this terminology is typically used to mock assets. As I've mentioned, sure, it's "wrong", and should be `mockETH` / `mockEth` / `mockEther`, but point is it is *consistently wrong around the codebase*, and `mockSomethingThatSoundsLikeAnAsset` (I doubt one would make the distinction between ETH the asset and Ethereum the network while visually parsing code) is what we expect to mock an asset.

By using the `chainAdapter` vernacular, you would remove the confusion.

Reply:
https://github.com/shapeshift/web/pull/4595/commits/0cc0dddcddf5cfa71691efc9782e74a74d34de92

Reply:
@lmyslinski this doesn't tackle the vernacular?

Reply:
https://github.com/shapeshift/web/commit/66611cd028621ddf2b118660f19c70ca0cd29887 does